{
  "affected": [
    {
      "database_specific": {
        "source": "https://storage.googleapis.com/debian-osv/debian-cve-osv/DEBIAN-CVE-2020-27802.json"
      },
      "ecosystem_specific": {
        "urgency": "unimportant"
      },
      "package": {
        "ecosystem": "Debian:11",
        "name": "upx-ucl",
        "purl": "pkg:deb/debian/upx-ucl?arch=source"
      },
      "ranges": [
        {
          "events": [
            {
              "introduced": "0"
            }
          ],
          "type": "ECOSYSTEM"
        }
      ],
      "versions": [
        "3.96-2",
        "3.96-2+deb11u1",
        "3.96-3",
        "4.2.2-1",
        "4.2.2-2",
        "4.2.2-3",
        "4.2.2-3~bpo12+1",
        "4.2.4-1",
        "4.2.4-1.1"
      ]
    },
    {
      "database_specific": {
        "source": "https://storage.googleapis.com/debian-osv/debian-cve-osv/DEBIAN-CVE-2020-27802.json"
      },
      "ecosystem_specific": {
        "urgency": "unimportant"
      },
      "package": {
        "ecosystem": "Debian:13",
        "name": "upx-ucl",
        "purl": "pkg:deb/debian/upx-ucl?arch=source"
      },
      "ranges": [
        {
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "4.2.2-1"
            }
          ],
          "type": "ECOSYSTEM"
        }
      ],
      "versions": []
    },
    {
      "database_specific": {
        "source": "https://storage.googleapis.com/debian-osv/debian-cve-osv/DEBIAN-CVE-2020-27802.json"
      },
      "ecosystem_specific": {
        "urgency": "unimportant"
      },
      "package": {
        "ecosystem": "Debian:14",
        "name": "upx-ucl",
        "purl": "pkg:deb/debian/upx-ucl?arch=source"
      },
      "ranges": [
        {
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "4.2.2-1"
            }
          ],
          "type": "ECOSYSTEM"
        }
      ],
      "versions": []
    }
  ],
  "details": "An floating point exception was discovered in the elf_lookup function in p_lx_elf.cpp in UPX 4.0.0 via a crafted Mach-O file.",
  "id": "DEBIAN-CVE-2020-27802",
  "modified": "2025-09-30T05:14:13.076467Z",
  "published": "2022-08-25T20:15:08Z",
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://security-tracker.debian.org/tracker/CVE-2020-27802"
    }
  ],
  "schema_version": "1.7.3",
  "severity": [
    {
      "score": "CVSS:3.1/AV:L/AC:L/PR:N/UI:R/S:U/C:N/I:N/A:H",
      "type": "CVSS_V3"
    }
  ],
  "upstream": [
    "CVE-2020-27802"
  ]
}